extensions/amp-consent/amp-consent.md

```diff
@@ -629,6 +629,7 @@ Join in on the discussion where we are discussing [upcoming potential features](
 -   Ogury : [Website](https://www.ogury.com/) - [Documentation](./cmps/ogury.md)
 -   OneTrust: [Website](https://www.onetrust.com/) - [Documentation](./cmps/onetrust.md)
 -   opencmp : [Documentation](./cmps/opencmp.md)
+-   Pubtech : [Documentation](./cmps/pubtech.md)
```

Comment:
Can you add your website: https://www.pubtech.ai/

Reply:
Yes, thanks!